cypress/support/commands.js

```diff
@@ -24,27 +24,27 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
+const imgUrl = Cypress.env('imgUrl');
```

Comment:
The variable `imgUrl` is declared but never used. If it's not needed, consider removing it to avoid unnecessary code.